src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -145,7 +135,17 @@ const ArticlePage = ({ pageData }) => {
       />
     ),
     timestamp: props =>
-      hasByline ? null : <Timestamp {...props} popOut={false} />,
+      hasByline ? (
+        <Byline {...props}>
```

Comment:
Playing around locally I think part of the problem is the `{...props}` will just be the `timestamp` blocks.

You can pass in `blocks={bylineContribBlocks}` into `Byline` and it will render, but it will appear above the media player.

I think there will still need to be some positioning logic in the BFF to move the `timestamp` below the first video/audio/image since its tied to `timestamp` now:

<img width="668" alt="Screenshot 2024-08-01 at 09 13 17" src="https://github.com/user-attachments/assets/ef65717d-3232-4c21-987f-e2fbe6e0df8c">